tox.ini

```diff
@@ -4,6 +4,7 @@ show_missing = True
 [coverage:run]
 branch = true
 parallel = true
+concurrency = multiprocessing
```

Comment:
This configuration change helps prevent coverage job errors caused by the usage of subprocess module. See details at https://github.com/pytest-dev/pytest-cov/issues/243